results/Pseudomonas_analysis_sample_lvl_sim_svcca_0.025.svg

```diff
@@ -0,0 +1,1962 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
Just to check that I'm understanding this correctly - these do have the same "valley" in the SVCCA scores for low number of partitions as the original experiments, right? It just comes back up so quickly that it looks like it's compressed to 0?

If so, is this what you expected (that it would take fewer partitions for the SVCCA score to come back up to ~1 when you decrease the noise magnitude)?

Reply:
Just to check that I'm understanding this correctly - these do have the same "valley" in the SVCCA scores for low number of partitions as the original experiments, right? It just comes back up so quickly that it looks like it's compressed to 0?
>>> yes this is correct.  I have added a zoomed in version of the figures so that its more clear what is happening

If so, is this what you expected (that it would take fewer partitions for the SVCCA score to come back up to ~1 when you decrease the noise magnitude)?
>>> Very good insight!  This is what i would expect to see.  Having a larger magnitude of variance would mean that we're adding very strong nonrandom noise to the original signal and so we need to have many more partitions in order to reduce the effect of each individual noise and regain our original signal.

Reply:
I think in part this is also driven by the choice of 10 PCs for SVCCA. With more PCs it would take longer to recover, but the overall patterns should be consistent.

Reply:
Yes thats true...with more PCs I suspect you'll be capturing more of the complexity that is in the structure (at least up until you start to hit that curse of dimensionality) and so it will be harder to match the structure.